src/device-api/v1.ts

```diff
@@ -204,6 +204,17 @@ router.patch('/v1/device/host-config', async (req, res) => {
 				log.warn('noProxy field must be an array of addresses');
 			}
 		}
+
+		// Validate dnsu2t config if it exists
+		const { dnsu2t } = req.body.network;
+		if (dnsu2t && typeof dnsu2t !== 'boolean' && typeof dnsu2t !== 'string') {
+			return res
+				.status(400)
+				.send(
+					'Expected dnsu2t to be a boolean or string, received: ' +
+						typeof dnsu2t,
+				);
+		}
```

Comment:
I didn't add validation for whether a dnsu2t string matches an [IP][:PORT] regex. Should we?

Reply:
Discussed that we aren't going to add validation as the rest of the proxy patch body isn't validated. It's up to user discretion to get their configs right